correlate.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python3
```

Comment:
I haven't worked with either numba or autocorrelation, so I will trust you on this ;)